src/Microsoft.DotNet.Tools.Builder/CompileContext.cs

```diff
@@ -125,19 +133,25 @@ private bool NeedsRebuilding(ProjectContext project, ProjectDependenciesFacade d
 
             if (!newInputs.Any())
             {
-                Reporter.Output.WriteLine($"\nProject {project.ProjectName()} was previously compiled. Skipping compilation.");
+                Reporter.Output.WriteLine($"Project {project.GetDisplayName()} was previously compiled. Skipping compilation.");
```

Comment:
I sent out a [PR](https://github.com/dotnet/cli/pull/1078) before I got a chance to read this comment. Feel free to close it the plan is to switch away from timestamps in entirety.